ldso/dynlink.c

```diff
@@ -105,7 +105,7 @@ struct symdef {
 
 static struct builtin_tls {
 	char c;
-        struct schedctx pt; /* was: pthread */
+        struct pthread pt;
```

Comment:
Does upstream have this indenting oddity?

Reply:
Thanks for spotting!